cognite/client/config.py

```diff
@@ -135,6 +135,7 @@ def __init__(
         self.credentials = credentials
         self.api_subversion = api_subversion or __api_subversion__
         self.base_url = (base_url or "https://api.cognitedata.com").rstrip("/")
+        self.auth_url = "https://auth.cognite.com"
```

Comment:
I think this would be better suited as a class variable on `OrgAPI`. Unless there is an actual need to override this?